pvanalytics/quality/irradiance.py

```diff
@@ -660,3 +660,267 @@ def calculate_component_sum_series(solar_zenith,
     return _fill_nighttime(component, component_sum_df,
                            fill_night_value,
                            solar_zenith, zenith_limit)
+
+
+def _upper_poa_global_limit_lorenz(aoi, solar_zenith, dni_extra):
+    r"""Function to calculate the upper limit of poa_global
+    """
+    # Changing aoi to 90 degrees when solar zenith is greater than 90 (sun
+    # below horizon) or aoi is greater than 90 (sun on the other side of
+    # the sensor/module's plane).
+    aoi = aoi.clip(lower=0, upper=90)
+
+    # Determining the upper limit
+    upper_limit = 0.9 * dni_extra * (cosd(aoi))**1.2 + 300
+
+    # Setting upper limit as 0 when solar zenith is > 90 (night time)
+    upper_limit[solar_zenith > 90] = 0
+
+    # Setting upper limit as undefined where solar_zenith is not available
+    upper_limit[solar_zenith.isna()] = np.nan
+
+    # Setting upper limit as undefined where aoi is not available
+    upper_limit[aoi.isna()] = np.nan
+
+    return upper_limit
+
+
+def _lower_limit_lorenz(solar_zenith, dni_extra):
+    r"""Function to calculate the lower limit of poa_global and ghi
+    """
+    # Setting the lower_limit at 0.
+    lower_limit = pd.Series(np.zeros(len(solar_zenith)),
+                            index=solar_zenith.index)
+
+    # Determining the lower limit when solar zenith is < 75
+    lower_limit = lower_limit.mask(solar_zenith < 75,
+                                   0.01 * dni_extra * cosd(solar_zenith))
+
+    # Setting lower limit as undefined where solar_zenith is not available
+    lower_limit[solar_zenith.isna()] = np.nan
+
+    return (lower_limit)
+
+
+def check_poa_global_limits_lorenz(poa_global, solar_zenith, aoi,
+                                   dni_extra=1367):
+    r"""Test for limits on POA global using the equations described in
+    Section 6.1 of [1]_
```

Comment:
The problem is that the first line of the docstring gets used as the description/blurb in the API listing, and if it includes a `[1]_`, that pointer gets naively reinterpreted in the context of the API listing page.  In `pvlib/pvlib-python` we don't have any references on the API listing page that it can point to, so we get build warnings about unknown references.  Here in `pvlib/pvanalytics` we do have some references on that page, so the footnote does link up with them, but the linking is wrong: `check_ghi_limits_lorenz`'s blurb links to the Long & Shi reference since that's what `[1]_` means on this page.

So yes the preference for not citing in the first sentence remains.  I think choice of theme is irrelevant; it's an autodoc/autosummary problem.